modules/cactus-web/src/SelectField/SelectField.story.tsx

```diff
@@ -13,6 +13,13 @@ export default {
 
 export const BasicUsage = (): React.ReactElement => (
   <div>
+    <SelectField
+      label={text('label', `Disabled`)}
```

Comment:
Change this to just "Disabled", right now it's overriding the default label on the active SelectField.
```suggestion
      label="Disabled"
```